modules/troubleshooting-network-observability-network-flow.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_content-type: PROCEDURE
+[id="configure-network-traffic-interfaces_{context}"]
+= Failing to see network flows from both `br-int` and `br-ex` interfaces
+
+`br-ex` and `br-int` are virtual bridge devices operated at OSI Layer 2. The eBPF agent works at Layers 3 and 4 (IP and TCP level). You can expect that the eBPF agent captures the network traffic passing through `br-ex` and `br-int`, when the network traffic is processed by other interfaces such as physical host or virtual pod interfaces. If you restrict the eBPF agent network interfaces to attach only to `br-ex` and `br-int`, you do not see any network flow.
```

Comment:
I'm not finding any guidance on whether it is OSI Layer or OSI layer. But I'd probably not capitalize _layer_.

Reply:
Addressed